<?xml version="1.0"?>
<launch>
  <arg name="input/vector_map" default="/map/vector_map"/>
  <arg name="param_path" default="$(find-pkg-share autoware_traffic_light_multi_camera_fusion)/config/traffic_light_multi_camera_fusion.param.yaml"/>
  <arg name="output/traffic_signals" default="/perception/traffic_light_recognition/traffic_signals"/>
  <arg name="camera_namespaces" default="[camera6, camera7]"/>

  <node pkg="autoware_traffic_light_multi_camera_fusion" exec="traffic_light_multi_camera_fusion_node" name="traffic_light_multi_camera_fusion" output="screen">
    <remap from="~/input/vector_map" to="$(var input/vector_map)"/>
    <remap from="~/output/traffic_signals" to="$(var output/traffic_signals)"/>
    <param from="$(var param_path)"/>
    <param name="camera_namespaces" value="$(var camera_namespaces)"/>
  </node>
</launch>
